src/main/resources/twilio-go/api_doc.mustache

```diff
@@ -14,11 +14,45 @@ Method | HTTP request | Description
 
 ## {{{operationId}}}
 
-> {{#returnType}}{{{returnType}}} {{/returnType}}{{{operationId}}}(ctx, {{#allParams}}{{#required}}{{paramName}}{{#hasMore}}, {{/hasMore}}{{/required}}{{/allParams}}{{#hasOptionalParams}}optional{{/hasOptionalParams}})
+> {{#returnType}}{{{.}}} {{/returnType}}{{{operationId}}}(ctx{{#pathParams}}, {{paramName}}{{/pathParams}}){{#allParams}}{{^isPathParam}}.{{vendorExtensions.x-export-param-name}}({{paramName}}){{/isPathParam}}{{/allParams}}.Execute()
```

Comment:
This is incorrect. The generated [method](https://github.com/twilio/twilio-go/blob/main/twilio/rest/api/v2010/api_default.go#L48) signature wouldn't match the [docs](https://github.com/twilio/twilio-go/blob/template-update/twilio/rest/api/v2010/docs/DefaultApi.md#CreateAccount).